src/jobs/Recruit.Vacancies.Jobs/TrainingTypes/StandardsAndFrameworksUpdater.cs

```diff
@@ -0,0 +1,115 @@
+
+                var standardsFromApi = standardsTask.Result.ToList();
+                var frameworksFromApi = frameworksTask.Result.ToList();
+
+                List<ApprenticeshipProgramme> newList = new List<ApprenticeshipProgramme>();
+                newList.AddRange(standardsFromApi);
+                newList.AddRange(frameworksFromApi);
+
+                var view = new ApprenticeshipProgrammeView
+                {
+                    ApprenticeshipProgrammes = newList
+                };
+                    
+                await UpdateQueryStore(view);
+
+                _logger.LogInformation("Inserted: {standardCount} standards and {frameworkCount} frameworks.", standardsFromApi.Count, frameworksFromApi.Count);
+            }
+            catch (AggregateException)
+            {
+                if (standardsTask.Exception != null)
+                {
+                    _logger.LogError(standardsTask.Exception, "Failed to get standards from api");
+                }
+
+                if (frameworksTask.Exception != null)
+                {
+                    _logger.LogError(frameworksTask.Exception, "Failed to get frameworks from api");
+                }
+
+                throw;
+            }
+        }
+
+        private async Task<IEnumerable<ApprenticeshipProgramme>> GetStandards()
+        {
+            _logger.LogTrace("Getting Standards from Apprentieships Api");
+
+            var retryPolicy = GetApiRetryPolicy();
+
+            var standards = await retryPolicy.ExecuteAsync(() => _standardsClient.GetAllAsync(), new Dictionary<string, object>() {{ "apiCall", "Standards" }});
+
+            return standards.FilterAndMapToApprenticeshipProgrammes();
+        }
+
+        private async Task<IEnumerable<ApprenticeshipProgramme>> GetFrameworks()
+        {
+            _logger.LogTrace("Getting Frameworks from Apprentieships Api");
+            
+            var retryPolicy = GetApiRetryPolicy();
+
+            var frameworks = await retryPolicy.ExecuteAsync(() => _frameworksClient.GetAllAsync(), new Dictionary<string, object>() {{ "apiCall", "Frameworks" }});
+            
+            return frameworks.FilterAndMapToApprenticeshipProgrammes();
+        }
+
+        private async Task UpdateQueryStore(ApprenticeshipProgrammeView view)
+        {
+            await _queryStore.UpdateStandardsAndFrameworksAsyc(view);
+        }
+        
+        private Polly.Retry.RetryPolicy GetApiRetryPolicy()
+        {
+            return Policy
+                    .Handle<Exception>()
+                    .WaitAndRetryAsync(new[]
+                    {
+                        TimeSpan.FromSeconds(1),
+                        TimeSpan.FromSeconds(2),
+                        TimeSpan.FromSeconds(4)
+                    }, (exception, timeSpan, retryCount, context) => {
+                        _logger.LogWarning($"Error connecting to Apprenticeships Api for {context["apiCall"]}. Retrying...attempt: {retryCount}");    
```

Comment:
Could we also log how long it is waiting for the next attempt. e.g. 2 seconds and the next go round 4 seconds